.github/workflows/mingw64-ci.yml

```diff
@@ -47,10 +47,6 @@ jobs:
           mkdir build64
           cd build64
           cmake -DSIMDJSON_BUILD_STATIC=ON -DSIMDJSON_COMPETITION=OFF -DSIMDJSON_GOOGLE_BENCHMARKS=OFF -DSIMDJSON_ENABLE_THREADS=OFF ..
-          cmake --build . --target basictests numberparsingcheck stringparsingcheck errortests integer_tests pointercheck --verbose
-          ctest . -R stringparsingcheck  --output-on-failure
-          ctest . -R numberparsingcheck  --output-on-failure
-          ctest . -R errortests  --output-on-failure
-          ctest . -R integer_tests  --output-on-failure
-          ctest . -R pointercheck  --output-on-failure
+          cmake --build . --target parse_many_test jsoncheck basictests numberparsingcheck stringparsingcheck errortests integer_tests pointercheck --verbose
+          ctest . -R "(parse_many_test|jsoncheck|basictests|stringparsingcheck|numberparsingcheck|errortests|integer_tests|pointercheck)"  --output-on-failure
```

Comment:
Would a --parallel on the build and test help make it go faster?

Reply:
The thing appears to freeze when the tests are too intense and I am preoccupied that --parallel may make things worse. In any case, it is quite fast... a few seconds now...